app/config.py

```diff
@@ -198,6 +198,11 @@ class Config(object):
             'schedule': crontab(),
             'options': {'queue': QueueNames.PERIODIC}
         },
+        'check-for-missing-rows-in-completed-jobs': {
+            'task': 'check-for-missing-rows-in-completed-jobs',
+            'schedule': crontab(minute='*/10'),
```

Comment:
I think that if it is not heavy on our resources, checking every 10 minutes should be fine. Some services use Notify to send emergency messages, so it is good if they are all processed as soon as possible I think.